tests/run.sh

```diff
@@ -204,6 +204,12 @@ for f in $tests; do
     # Send the test file to awesome.
     cat $f | DISPLAY=$D "$AWESOME_CLIENT" 2>&1
 
+    # Kill awesome after 1 minute (e.g. with errors during test setup).
+    (sleep 60; if kill -0 $awesome_pid 2>/dev/null; then
+        echo "Killing (stale?!) awesome (PID $awesome_pid) after 10 seconds."
+        kill $awesome_pid
+      fi) &
+
```

Comment:
Perhaps it would be easier to do `echo 'require("gears.timer").start_new(60, function() awesome.quit() end)' | DISPLAY=$D "$AWESOME_CLIENT"`. Less code and less likely that the script kills a random, other PID.